specificatie/verblijfplaats.yaml

```diff
@@ -49,11 +49,16 @@ components:
       type: string
       pattern: ^[a-zA-Z0-9 \-]{1,4}$
       example: "bis"
-    KorteNaam:
+    KorteStraatnaam:
       description: |
-        De officiële openbareruimtenaam uit de Basisregistratie Gebouwen en Adressen (BAG) of een verkorte versie.
+        De verkorte versie van de officiële openbareruimtenaam uit de Basisregistratie Gebouwen en Adressen (BAG) of de straatnaam van een niet-BAG adres.
```

Comment:
Voorstel: De officiële openbareruimtenaam uit de Basisregistratie Gebouwen en Adressen (BAG), zo nodig verkort tot maximaal 24 tekens, of de straatnaam van een niet-BAG adres.

Reply:
is "openbareruimtenaam" 1 woord?

Reply:
> is "openbareruimtenaam" 1 woord?

"naam van de openbare ruimte" van gemaakt.